src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstSide;
+    private double secondSide;
+
+    public Rectangle(String color, double firstSide, double secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    public void setFirstSide(double firstSide) {
+        this.firstSide = firstSide;
+    }
+
+    public void setSecondSide(double secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    public double getFirstSide() {
+        return firstSide;
+    }
+
+    public double getSecondSide() {
+        return secondSide;
+    }
+
+    @Override
+    public double area() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, " + "area: " + area() + " sq. units, " + "first side: " + firstSide + " units, second side: " + secondSide + " units, " + "color: " + getColor());
```

Comment:
Use only English in messages. The message in the `draw` method contains a mix of code and English which might be confusing. Keep the message purely in English for consistency.